Frends.PowerShell.RunScript/Frends.PowerShell.RunScript/Frends.PowerShell.RunScript.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+	<TargetFrameworks>net6.0</TargetFrameworks>
+	<ImplicitUsings>enable</ImplicitUsings>
+	<IncludeSource>true</IncludeSource>
+	<AssemblyName>Frends.PowerShell.RunScript</AssemblyName>
+	<RootNamespace>Frends.PowerShell.RunScript</RootNamespace>
+    
+	<Version>1.0.1</Version>
+	<Authors>Frends</Authors>
+	<Copyright>Frends</Copyright>
+	<Company>Frends</Company>
+	<Product>Frends</Product>
+	<PackageTags>Frends</PackageTags>
+	<PackageLicenseExpression>MIT</PackageLicenseExpression>
+	<GenerateDocumentationFile>true</GenerateDocumentationFile>
+	<Description>Task for running scripts on PowerShell. Requires PowerShell 7.0 to work.</Description>
+	<PackageProjectUrl>https://github.com/FrendsPlatform/Frends.PowerShell</PackageProjectUrl>
```

Comment:
This should be `https://frends.com/`, as in https://github.com/FrendsPlatform/Frends.Excel/blob/main/Frends.Excel.ConvertToCSV/Frends.Excel.ConvertToCSV/Frends.Excel.ConvertToCSV.csproj#L19

Reply:
Fixed